<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f3957c
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="captureButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>730</y>
      <width>446</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>115</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushBuuton{
	border: 1px solid;
	border-radius: 57px;
	border-color: blue;
}

QPushButton:enabled {
	border-radius: 57px;
	background-color: #0021A5;
	color: white;
}

QPushButton:pressed {
	background-color: #6b87f1;
	color: #fffffe;
}
QPushButton:hover:!pressed {
	background-color: #E1f4FF;
	color: #0c2f70
}</string>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recaptureButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>730</y>
      <width>446</width>
      <height>115</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushBuuton{
	border: 1px solid;
	border-radius: 50px;
	border-color: blue;
}

QPushButton:enabled {
	border-radius: 57px;
	background-color: #0021A5;
	color: white;
}

QPushButton:pressed {
	background-color: #6b87f1;
	color: #fffffe;
}
QPushButton:hover:!pressed {
	background-color: #E1f4FF;
	color: #0c2f70
}</string>
    </property>
    <property name="text">
     <string>Recapture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>730</y>
      <width>446</width>
      <height>115</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushBuuton{
	border: 1px solid;
	border-radius: 50px;
	border-color: blue;
}

QPushButton:enabled {
	border-radius: 57px;
	background-color: #0021A5;
	color: white;
}

QPushButton:pressed {
	background-color: #6b87f1;
	color: #fffffe;
}
QPushButton:hover:!pressed {
	background-color: #E1f4FF;
	color: #0c2f70
}</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="objectNameLable">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Object Identifier</string>
    </property>
   </widget>
   <widget class="QLabel" name="definitionLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>801</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Definition</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>recaptureButton</zorder>
   <zorder>continueButton</zorder>
   <zorder>objectNameLable</zorder>
   <zorder>definitionLabel</zorder>
   <zorder>imageLabel</zorder>
   <zorder>captureButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
